Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -214,14 +214,12 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
 
     let hideWhenReplay = Signal.merge(
       project.mapConst(true),
-      event.map { !$0.liveNow },
-      self.showErrorAlert.mapConst(true)
```

Comment:
The screenshot tests that were failing were because of this line - previously we would fetch the `LiveStreamEvent` asynchronously and if it failed we would keep some views hidden. Because this is now available immediately at initialisation this sort of thing is no longer necessary as errors are now only shown for live stream playback issues.